hack/make/.build-deb/control

```diff
@@ -13,7 +13,7 @@ Recommends: aufs-tools,
             git,
             xz-utils,
             ${apparmor:Recommends}
-Conflicts: docker (<< 1.5~), docker.io, lxc-docker, lxc-docker-virtual-package
+Conflicts: docker (<< 1.5~), docker.io, lxc-docker, lxc-docker-virtual-package, docker-engine-cs
```

Comment:
A decision was made to not actually have different package names, but instead just let the version string speak for itself. So depending on the repository you have added to your package manager, you would get either:

```bash
$ docker --version
Docker version 1.9.0, build 76d6bc9
```

or
```bash
$ docker --version
Docker version 1.9.0-cs1-rc1, build e1417a4
```

Plus, they do actually conflict. You can't install the open-source version and the CS version at the same time. The second install will clobber `$(which docker)` the first one installed. And at this point, docker-engine-cs is basically an old package that users might have installed that does conflict with current things, just like the other names here (docker.io, lxc-docker).